internal/nri/sandbox_linux.go

```diff
@@ -0,0 +1,38 @@
+//go:build linux
+// +build linux
+
+/*
+   Copyright The containerd Authors.
```

Comment:
Is this a copy-paste from containerd code?

Reply:
Well, I first added a (structurally very similar) implementation for containerd, then bootstrapped this one from the same code... and forgot to nuke the headers from files which didn't need any touching. I'd rather say I wrote both at the same time... Thanks for spotting it. Removed the header.